client/util.go

```diff
@@ -78,15 +80,17 @@ func shuffleStrings(list []string) {
 
 // persistState is used to help with saving state
 func persistState(path string, data interface{}) error {
```

Comment:
This is unused and can be removed now right?

If it's still called from tests perhaps move to a `_test.go` file?